src/pyodide-py/_pyodide/_core.py

```diff
@@ -0,0 +1,76 @@
+# type: ignore
+
+from typing import Any, Callable
+
+# All docstrings for public `core` APIs should be extracted from here. We use
+# the utilities in `docstring.py` and `docstring.c` to format them
+# appropriately.
+
+# Sphinx uses __name__ to determine the paths and such. It looks better for it
+# to refer to e.g., `pyodide.JsProxy` than `_pyodide._core.JsProxy`.
+__name__ = "pyodide"
```

Comment:
As long as it doesn't break the import mechanism otherwise..

Reply:
Maybe I should restore it at the bottom of the file.